### PR TITLE
First page/create first page by html and css

### PR DESCRIPTION
**Pull request by MalakAyyash**

### CODE REVIEW

File: main.html [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    <title>Birzeit University</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/js/bootstrap.bundle.min.js"></script>
+    <script src="https://kit.fontawesome.com/364e2eed3f.js" crossorigin="anonymous"></script>
+   
+</head>
+<!-- ********************************************************************** -->
+<body class="text-white">
+
```
Comment (MalakAyyash) [inline main.html:15]: please revmove extra lines

Comment (MuathYHKefayah) [inline main.html]: remove an extra lines please

Comment (MuathYHKefayah) [reply main.html]: test

**Review by MuathYHKefayah — CHANGES_REQUESTED** [review]

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+}
+  @media only screen and (max-width: 500px) {
+  .texts p{
+    padding-left: 10%;
+  
+  } 
+  .texts h1{
+    font-size: 70px;
+  }
+}
```
Comment (MuathYHKefayah) [inline css/style.css:240]: Great work!

File: index.html [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+        </li>
+      </ul>
+    </div>
+    <div class="second-footer p-1 pt-2">
+      <p>&copy; Malak Ayyash.All Rights Reserved. </p>
+    </div>
+  </footer>
+</body>
+</header>
+</html>
```
Comment (MuathYHKefayah) [inline index.html:207]: Well done!

**Review by MuathYHKefayah — APPROVED** [review]

**Pull request merged**